src/App.vue

```diff
@@ -127,7 +130,7 @@ export default {
 
   computed: {
     ...mapGetters('auth', ['isAuthenticated']),
-    ...mapState('app', ['isLoaded']),
+    ...mapState('app', ['isLoaded', 'isRefreshing']),
```

Comment:
Is `isRefreshing` left over from something you removed? I don't see it defined or used anywhere.